spectrum/autism_prevalence_map/templates/autism_prevalence_map/footer.html

```diff
@@ -0,0 +1,171 @@
+{% load svg %}
+
+<footer class="bg-navy text-white">
+    <section
+        class="max-w-400 mx-auto grid grid-cols-4 gap-4 px-4 py-4 lg:grid-cols-12 lg:gap-8 lg:py-8 lg:px-22 2xl:px-17.5">
+        <div class="mt-6 col-span-4 flex flex-col justify-between lg:col-span-12 lg:mt-12 lg:flex-row">
+            <div
+                class="flex flex-col font-sans font-bold text-2xs text-white items-center uppercase lg:w-73 lg:items-start">
+                <div class="menu-footer-left-container">
+                    <ul class="grid gap-4 mb-4 tracking-extra-wide text-center lg:text-start">
+                        <li>
+                            <a href="https://www.thetransmitter.org/./news/">News</a>
+                        </li>
+                        <li>
+                            <a href="https://www.thetransmitter.org/./perspectives/">Perspectives</a>
+                        </li>
+                        <li>
+                            <a href="https://www.thetransmitter.org/./features/">Features</a>
+                        </li>
+                        <li>
+                            <a href="https://www.thetransmitter.org/./profiles/">Profiles</a>
+                        </li>
+                        <li>
+                            <a href="https://www.thetransmitter.org/./beyond-the-bench/">Beyond the bench</a>
+                        </li>
+                        <li>
+                            <a href="https://www.thetransmitter.org/./podcasts/">Podcasts</a>
+                        </li>
+                        <li>
+                            <a href="https://www.thetransmitter.org/./videos/">Videos</a>
+                        </li>
+                        <li>
+                            <a href="https://www.thetransmitter.org/./books/">Books</a>
+                        </li>
+                        <li>
+                            <a href="https://www.thetransmitter.org/transmitter-books/"><em>The Transmitter</em> books</a>
+                        </li>
+                    </ul>
+                </div> <a href="https://www.thetransmitter.org/spectrum/"
+                    class="flex text-light-navy-2 w-25 gap-2 items-center hover:text-light-red lg:w-unset">
+                    <svg xmlns="http://www.w3.org/2000/svg" class="w-5 h-5 lg:h-3.5 lg:w-3.5" width="13" height="14"
```

Comment:
Can you please put the svg code into a file and put it under folder **img** as mentioned above. Here you can use Django template tag like the following example as you did at the code below:
` {% svg 'spectrum-logo' %}`